rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2087,7 +2090,8 @@ private Integer createFederation(boolean dryRun) {
         FederationConstants federationConstants = bridgeConstants.getFederationConstants();
         provider.getFederationElection(federationConstants.getFederationChangeAuthorizer()).clear();
 
-        return 1;
+        logger.info("[createFederation] Federation created successfully.");
```

Comment:
```suggestion
        logger.info("[createFederation] Pending federation created successfully.");
```
We could also rename this method to `createPendingFederation`, I think it's more accurate

Reply:
good one. Thanks!

Reply:
Maybe do the same with `commitFederation`? Rename it to `commitPendingFederation`? wdyt?

Reply:
Interesting, I'm not sure to be honest.

Once you commit the pending federation it is no longer pending, it is a federation. And in the future it will be a proposed federation. So depending on which side you look at this it is a pending or a proposed fed.

On the other hand, the event emitted is called `commit_federation`.

Reply:
for the way i see it, when you call the method its still a pending federation. it's after the implementation of the method that it becomes a federation / proposed federation